### DISCUSSION TITLE
Instead of battery Peltier element

### DISCUSSION THREAD

**Discussion by He-Energy** [joeycastillo/Sensor-Watch#125] [Ideas]
Hello, I've been a fan of the Casio watch since childhood and have only recently come across the mega project. 

Wanted to make a suggestion about energy supply. 

A Peltier element generates a small voltage via a warm and a cold side through the Seebeck effect. 
Which could be sufficient to operate the clock.

 The warm side can be the wrist and the cold side the outside air Whereby it must have a certain temperature difference

**Comment by bademux** [comment]
Nice idea, it requires math do be done, and I believe some (super)capacitor.
